src/appengine/handlers/cron/oss_fuzz_build_status.py

```diff
@@ -139,7 +139,7 @@ def file_bug(issue_tracker, project_name, build_id, ccs, build_type):
   logs.log('Filing bug for new build failure (project=%s, build_type=%s, '
            'build_id=%s).' % (project_name, build_type, build_id))
 
-  issue = issue_tracker.new_issue()
+  issue = issue_tracker.new_issue('job_type')
```

Comment:
I left a comment where this function is defined, but this usage feels a bit hacky to me. I think it should be possible to remove the need for it.

Reply:
Agreed - this will get rolled back to the current mode of calling because the only reason I made the change was to be able to call `data_handler.get_value_from_job_definition` and that's resolved by using PROJECT_NAME instead.